governance/mission-values.md

```diff
@@ -16,29 +15,74 @@ Science is now data-driven. And as such, data science supports scientists
 processing the data that they need to address critical questions.
 
 We value open science workflows because they allow scientists to build upon each
-others work rather than recreating already developed workflows. Building on
-top of existing workflows supports science developing and evolving at a quicker pace.
+other's work rather than recreating already developed workflows. Building on
+top of existing workflows accelerates science as it prevents scientists from needing to rebuild existing workflows from the ground up.
 
 ## Why is (free) and open source so important?
 
-Open source software refers to the tools that scientists (and others) use to
+Open source software refers to the tools scientists (and others) use to
 process data. Free and open software is critical to science as it allows
-scientists to freely access the tools that they need to process their data.
-If the tools are free and open, then it lowers barriers for others to use them
-without having to pay costly subscription fees.
+scientists to access the tools they need to process their data freely.
+If the tools are free and open, they lower barriers for others to use them
+without paying costly subscription fees.
 
-This free and open source software is critical to supporting development of
+This free and open-source software is critical to supporting the development of
```

Comment:
I don't think "open source" is generally hyphenated.

Reply:
i've had this question for a while and googled this topic. I'll keep it unhyphenated, but in some places like the [microsoft style guide](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/o/open-source), it's hyphenated!! Let's decide that as an org we don't hyphenate it. 😆 if I add it to grammarly it will remind me not to. I have no opinion here.